### PR TITLE
Support Python plugin

### LINKED ISSUE #40: Support Python plugin

### PR DESCRIPTION
**Pull request by aooohan**

### CODE REVIEW

**Comment by aooohan** [comment]
Only vfox >= 0.2.3 can use the current plugin

File: python/python.lua [hunk 1]
```diff
@@ -0,0 +1,224 @@
...
+    --- Plugin name
+    name = "python",
+    --- Plugin author
+    author = "aooohan",
+    --- Plugin version
+    version = "0.0.1",
+    description = "For Windows, only support >=3.5.0, but no restrictions for unix-like",
+    -- Update URL
+    updateUrl = "https://github.com/aooohan/version-fox-plugins/blob/main/python/python.lua",
+    minRuntimeVersion = "0.2.2",
```
Comment (baymax55) [inline python/python.lua]: Does this mean it will work on vfox 0.2.2 and later?

Comment (aooohan) [reply python/python.lua]: Oops, I forgot to change it here, it should be 0.2.3

**Comment by aooohan** [comment]
I have done the test on Windows that was great. After vfox 0.2.3 is released, this pr will be merged.

Some  limitations on Windows:
1. only the target version that provides the exe installer is supported. e.g. 3.11.0
2. only support >= 3.5.0

For 1), it can be enhanced later to support the installation of msi, zip, and web-installer.

**Pull request merged**